target/linux/ramips/dts/mt7628an_hiwifi_hc5611.dts

```diff
@@ -0,0 +1,29 @@
+#include "mt7628an_hiwifi_hc5x61a.dtsi"
```

Comment:
It is recommended to add GPL license here
```
// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```
And please remove new blank line in #L21